### PR TITLE
Extend Support for other JSON encoders like yajl-ruby

### PR DESCRIPTION
**Pull request by ritikesh**

### CODE REVIEW

File: lib/blueprinter/configuration.rb [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-    attr_accessor :generator
+    attr_accessor :generator, :method
 
     def initialize
       @generator = JSON
+      @method = :generate
+    end
+
+    def jsonify(blob)
+      generator.send(method, blob)
```
Comment (philipqnguyen) [inline lib/blueprinter/configuration.rb]: Thank you for doing this! Just one minor request. Can we use `public_send` over `send`?

Comment (ritikesh) [reply lib/blueprinter/configuration.rb]: Thanks for getting back on this. I've changed it to use public_send. Let me know if you need anything else.

**Comment by ritikesh** [comment]
@philipqnguyen Thanks for approving the PR. Any updates on when this will be merged?

**Comment by philipqnguyen** [comment]
I don't do the releases anymore, but @mcclayton does!

**Comment by mcclayton** [comment]
Thank you so much for the contribution @ritikesh! These changes look good to me and I'll merge these now, I will try to draft the next blueprinter release later today 👍

**Pull request merged**